alluxio/controllerserver.go

```diff
@@ -0,0 +1,109 @@
+package alluxio
+
+import (
+    "crypto/sha1"
+    "encoding/hex"
+    "github.com/container-storage-interface/spec/lib/go/csi"
+    "github.com/golang/glog"
+    csicommon "github.com/kubernetes-csi/drivers/pkg/csi-common"
+    "golang.org/x/net/context"
+    "google.golang.org/grpc/codes"
+    "google.golang.org/grpc/status"
+    "io"
```

Comment:
nit: Please separate the imports. I think the format tool (We use `godelw format` but not sure what you typically refer to) will put `"io"` and `"strings"` at the top and have an extra row before the rest of imports. Please ignore this if that's what your format tool returns. @Xenorith has more say on this.

Reply:
more importantly the imports across each file should follow the same pattern. this one organizes all of them by alphabetical order but the subsequent ones group native imports first, then vendored imports. i have a slight preference for grouping but i'm happy as long as there's consistency across files.